android/src/main/java/com/horcrux/svg/SvgViewManager.java

```diff
@@ -87,13 +87,15 @@ public boolean needsCustomLayoutForChildren() {
     return true;
   }
 
-  @ReactProp(name = "tintColor")
-  public void setTintColor(SvgView node, @Nullable Dynamic tintColor) {
+  @ReactProp(name = "tintColor", customType = "Color")
```

Comment:
This also simplifies the parsing of color so we don't need to use `ColorPropConverter` for the `Dynamic` type that could have been provided.